moped-editor/src/views/projects/projectView/ProjectView.js

```diff
@@ -554,7 +554,7 @@ const ProjectView = () => {
                             data={data}
                             error={error}
                             refetch={refetch}
-                            projectName={data.moped_project[0].project_name}
+                            projectName={data.moped_project[0].project_name_full}
```

Comment:
this is what is shown on the map tab

Reply:
This made me think of the title that shows in the browser tab. Can't remember where that is set, and I don't know if it is important enough to change tbh. 🤷

Reply:
good catch!! and ive updated it